tools/do_gpt.sh

```diff
@@ -62,19 +62,28 @@ else
   TIME=
 fi
 
-gpart create -s gpt ${unit}
-gpart add -t freebsd-boot -b 40 -l boot -s 472 ${unit}
-gpart bootcode -b ${BOOTDIR}/pmbr -p ${BOOTDIR}/gptboot -i 1 ${unit}
-if [ -n "${EFIIMG}" -a -f "${EFIIMG}" ]; then
-  gpart add -t efi -s 2m ${unit}
-  ${TIME} dd if=${EFIIMG} of=/dev/${unit}p2 bs=128k
-fi
-gpart add -t freebsd-ufs -l rootfs ${unit}
-${TIME} makefs -B little ${TMPIMG} ${FSPROTO}
-if [ -f ${EFIIMG} ]; then
-  ${TIME} dd if=${TMPIMG} of=/dev/${unit}p3 bs=128k
+if [ `uname -m` = 'arm64' ]; then
```

Comment:
This is strange, the build should be host-independent, only target should impact any changres.

Reply:
For aarch64, there's different layout of partitions and there's no need for installing bootcode - I tried to make my changes in that way, they won't touch the existing logic, but feel free to propose/change it any way you feel it should be.